### ISSUE TITLE
Make l10n possible

### ISSUE THREAD

**Issue by vinzv** [hannesmannerheim/quitim#12]
Would be good to have a separate json/po/whatever file to translate all strings needed.

**Comment by hannesmannerheim** [comment]
definitely. gnu social has support for localisation of plugins, although i don't know exactly how it works

**Comment by vinzv** [comment]
GS does it with GNU po files and compiling them. Maybe the easiest way to get quitim translated is the same as you did with Qvitter?

**Comment by hannesmannerheim** [comment]
i think for quitim it will be best to do it in the same way as gnusocial does it
